### PR TITLE
update metadata and inflation

### PR DESCRIPTION
**Pull request by Leouarz**

### CODE REVIEW

**Comment by Leouarz** [comment]
Small note: explorer is right now on a really old deployment, that's why ppl cannot transfer token yet @kroos47 @prabal-banerjee

File: packages/apps-config/src/api/constants.ts [hunk 1]
```diff
@@ -36,3 +36,5 @@ export const NFTMART_GENESIS = '0xfcf9074303d8f319ad1bf0195b145871977e7c375883b8
 export const CERE_NETWORK_GENESIS = '0x81443836a9a24caaa23f1241897d1235717535711d1d3fe24eae4fdc942c092c';
 
 export const CERE_NETWORK_TESTNET_GENESIS = '0x42b9b44b4950b6c1edae543a7696caf8d0a160e9bc0424ab4ab217f7a8ba30dc';
+
+export const KATE_NETWORK_TESTNET_GENESIS = '0xd12003ac837853b062aaccca5ce87ac4838c48447e41db4a3dcfb5bf312350c6'
```
Comment (prabal-banerjee) [inline packages/apps-config/src/api/constants.ts:40]: Is this relevant anymore now that the testnet is being sunset?

**Review by prabal-banerjee — APPROVED** [review]

File: packages/apps-config/src/api/constants.ts (hunk 1 above)
Comment (Leouarz) [reply packages/apps-config/src/api/constants.ts:40]: It's ok, it'll be here and update when we have genesis on new testnet

**Review by kroos47 — APPROVED** [review]

**Comment by tdelabro** [comment]
Thx for fixing inflation @Leouarz 📈

**Pull request merged**